srcpkgs/emptty/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'emptty'
+pkgname=emptty
+version=0.1.0
+revision=1
+archs="x86_64"
```

Comment:
Why is this `x86_64` only? With `build_style=go`, I can build this on `aarch64` as well. (See the comment on the next line.)